### PR TITLE
add more code owners

### PR DESCRIPTION
**Pull request by kevinintel**
## Description

Add more code owners

## Issues

n/a

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [Y] Others (enhancement, documentation, validation, etc.)

## Dependencies

n/a

## Tests

n/a

### CODE REVIEW

**Review by lvliang-intel — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,9 +1,12 @@
+/AgentQnA/ xuhui.ren@intel.com
 /AudioQnA/ sihan.chen@intel.com
 /ChatQnA/ liang1.lv@intel.com
 /CodeGen/ liang1.lv@intel.com
 /CodeTrans/ sihan.chen@intel.com
 /DocSum/ sihan.chen@intel.com
-/FaqGen/ letong.han@intel.com
+/DocIndexRetriever/ xuhui.ren@intel.com
```
Comment (hshen14) [inline .github/CODEOWNERS:7]: Can we add Chendi after Xuhui?

**Comment by dbkinder** [comment]
@kevinintel @hshen14 This PR has errors as noted by GitHub:
(there shouldn't be a comma, only a space between owners)
```
Invalid owner on line 7:  
…Retriever/ xuhui.ren@intel.com, chendi.xue@intel.com
Unknown owner on line 12: make sure a user with the email address hoong.tee.yeoh@intel.com exists and has write access to the repository
/ProductivitySuite/ hoong.tee.yeoh@intel.com
```

**Comment by kevinintel** [comment]
More and more developers will contribute and create new folder.
They may not have write permission.

**Review by hshen14 — APPROVED** [review]

**Review by ftian1 — APPROVED** [review]

**Review by dbkinder — COMMENTED** [review]
@ftian1 @kevinintel GitHub is complaining that this CODEOWNERS file contains errors.

Also, by putting /*/ at the end of the CODEOWNERS, it will assign liang1.lv@intel.com as the owner of all directories (last matching pattern wins).  You should put this at the top of the file.  We're also missing a pattern for files in the repo root (e.g, for README.md, docker_images_list.md, pyproject.toml, etc.)

**Pull request merged**